s2i-dotnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Source-to-Image
+
+[Source-to-Image (S2I)](https://github.com/openshift/source-to-image)
+is a toolkit and workflow for building reproducible container images
```

Comment:
a workflow

Reply:
Is 'and' correct here? Should it have been 'or' instead? As in - a toolkit or a workflow